src/org/exist/xquery/parser/DeclScannerTokenTypes.java

```diff
@@ -193,62 +193,63 @@
 	int LITERAL_node = 187;
 	int LITERAL_attribute = 188;
 	int LITERAL_comment = 189;
-	// "processing-instruction" = 190
-	// "document-node" = 191
-	int LITERAL_document = 192;
-	int HASH = 193;
-	int SELF = 194;
-	int XML_COMMENT = 195;
-	int XML_PI = 196;
-	int LPPAREN = 197;
-	int RPPAREN = 198;
-	int AT = 199;
-	int PARENT = 200;
-	int LITERAL_child = 201;
-	int LITERAL_self = 202;
-	int LITERAL_descendant = 203;
-	// "descendant-or-self" = 204
-	// "following-sibling" = 205
-	int LITERAL_parent = 206;
-	int LITERAL_ancestor = 207;
-	// "ancestor-or-self" = 208
-	// "preceding-sibling" = 209
-	int INTEGER_LITERAL = 210;
-	int DOUBLE_LITERAL = 211;
-	int DECIMAL_LITERAL = 212;
-	// "schema-element" = 213
-	int END_TAG_START = 214;
-	int QUOT = 215;
-	int APOS = 216;
-	int QUOT_ATTRIBUTE_CONTENT = 217;
-	int ESCAPE_QUOT = 218;
-	int APOS_ATTRIBUTE_CONTENT = 219;
-	int ESCAPE_APOS = 220;
-	int ELEMENT_CONTENT = 221;
-	int XML_COMMENT_END = 222;
-	int XML_PI_END = 223;
-	int XML_CDATA = 224;
-	int LITERAL_collection = 225;
-	int LITERAL_validate = 226;
-	int XML_PI_START = 227;
-	int XML_CDATA_START = 228;
-	int XML_CDATA_END = 229;
-	int LETTER = 230;
-	int DIGITS = 231;
-	int HEX_DIGITS = 232;
-	int NMSTART = 233;
-	int NMCHAR = 234;
-	int WS = 235;
-	int XQDOC_COMMENT = 236;
-	int EXPR_COMMENT = 237;
-	int PREDEFINED_ENTITY_REF = 238;
-	int CHAR_REF = 239;
-	int S = 240;
-	int NEXT_TOKEN = 241;
-	int CHAR = 242;
-	int BASECHAR = 243;
-	int IDEOGRAPHIC = 244;
-	int COMBINING_CHAR = 245;
-	int DIGIT = 246;
-	int EXTENDER = 247;
+	// "namespace-node" = 190
+	// "processing-instruction" = 191
+	// "document-node" = 192
+	int LITERAL_document = 193;
```

Comment:
the values starting with LITERAL_document  shifted +1 , is this dangerous for existing DBX files?